cargo-concordium/src/build.rs

```diff
@@ -2,6 +2,7 @@ use ansi_term::{Color, Style};
 use anyhow::Context;
 use cargo_toml::Manifest;
 use concordium_contracts_common::*;
+use rand::{thread_rng, RngCore};
```

Comment:
We generally use `Rng` as a constraint, not `RngCore`. `Rng` adds a few more helpers on top, and there is no loss in requiring Rng since there is a generic implementation of Rng for anything t hat implements RngCore.

Reply:
Done.